src/tool_parsecfg.c

```diff
@@ -39,7 +39,18 @@
 #define ISSEP(x,dash) (!dash && (((x) == '=') || ((x) == ':')))
 
 static const char *unslashquote(const char *line, char *param);
-static char *my_get_line(FILE *fp);
+
+#define MAX_CONFIG_LINE_LENGTH (100*1024)
+/*
+ * Reads a line from the given file, ensure the buffer is NUL terminated.
+ * Returns TRUE on error.
+ */
+static bool my_get_line(FILE *fp, char *buf, int buflen)
+{
+  if(!fgets(buf, buflen, fp))
+    return TRUE; /* error */
```

Comment:
```suggestion
    return FALSE; /* error */
```
I suggest keep with standard return values for bool functions that rather than try to imitate the old function's behavior.

Another thing I notice is the old version clipped the line terminator and this one doesn't.

Reply:
> standard return values for bool functions

I wasn't really aware we had such. But sure, switching the return value is easy!

> the old version clipped the line terminator and this one doesn't.

Yeah I discovered our code already deals with them anyway so it was an unnecessary operation in  the load function.